crates/sui-json-rpc/src/read_api.rs

```diff
@@ -540,10 +540,12 @@ impl ReadApiServer for ReadApi {
             Some(self.state.get_executed_transaction(digest).await.tap_err(
                 |err| debug!(tx_digest=?digest, "Failed to get transaction: {:?}", err),
             )?);
+        println!("fetched transaction: {:?}", transaction);
 
         // the input is needed for object_changes to retrieve the sender address.
         if opts.require_input() {
             temp_response.transaction = transaction;
+            println!("temp_response.transaction: {:?}", temp_response.transaction);
```

Comment:
nit: log the digest

Reply:
It's also possible that the error happens in the conversion from `TransactionResponse` to `SuiTransactionResponse`, can you add some info! log to there as well?